browserprocess/register_test.go

```diff
@@ -1,57 +0,0 @@
-package browserprocess
```

Comment:
This became a bit meaningless after removing `unregister`. Testing this is a little tricky, but let me try again 🤔